.travis.yml

```diff
@@ -21,7 +21,8 @@ script:
       chmod a+x $HOME/bin/aws-iam-authenticator;
       export PATH=$PATH:$HOME/bin;
       echo "$KUBECONFIG_AWS" > kubeconfig_aws;
-      make e2e-test -e KUBECONFIG=`pwd`/kubeconfig_aws;
+      echo "$E2E_TEST_CONFIG_DATA" > e2e-test-config.json;
```

Comment:
Is the config file supposed to be in YAML or JSON? There is some JSON references but the unmarshall function and the config file example are using YAML

Reply:
Both json and yaml are supported. But good point, I just added .json suffix because I use JSON format in my travis setup. I removed the .json suffix